tests/ndc/ndc_integration_test.go

```diff
@@ -402,6 +403,80 @@ func (s *nDCIntegrationTestSuite) TestEmptyVersionAndNonEmptyVersion() {
 	)
 }
 
+func (s *nDCIntegrationTestSuite) TestReplicateWorkflowState_PartialReplicated() {
+
+	s.setupRemoteFrontendClients()
+	workflowID := "ndc-single-branch-test" + uuid.New()
```

Comment:
plz make the workflow ID unique, e.g. by naming it `replicate-workflow-state-partially-replicated`